### PR TITLE
Feature/f header

### PR DESCRIPTION
**Pull request by tathian**
Please check my branch and merge it?  thank you.

### CODE REVIEW

File: Rose/database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -23,6 +23,14 @@ public function up()
             $table->rememberToken();
             $table->timestamps();
         });
+
+        DB::table('users')->insert([
```
Comment (nhanhvasnet) [inline Rose/database/migrations/2014_10_12_000000_create_users_table.php]: Why seed data in here?

File: Rose/routes/web.php [hunk 2]
```diff
@@ -25,4 +33,5 @@
 Route::get('forgot-pass', 'PageController@getForgotPassword');
 
 // form change pasword
-Route::get('change-pass', 'PageController@getChangePassword');
\ No newline at end of file
+Route::get('change-pass', 'PageController@getChangePassword');
+>>>>>>> 0aff541e0d5d790e0777227621c5046529286c47
```
Comment (nhanhvasnet) [inline Rose/routes/web.php]: What is this?

File: Rose/routes/web.php [hunk 3]
```diff
@@ -15,6 +15,14 @@
     return view('welcome');
 });
 
+<<<<<<< HEAD
```
Comment (nhanhvasnet) [inline Rose/routes/web.php]: What is this?

File: Rose/routes/web.php [hunk 4]
```diff
@@ -15,6 +15,14 @@
...
+Route::get('header', function () {
+    return view('header');
+});
+Route::get('footer', function () {
+    return view('footer');
+});
+=======
```
Comment (nhanhvasnet) [inline Rose/routes/web.php]: What is this?

File: Rose/database/migrations/2014_10_12_000000_create_users_table.php (hunk 1 above)
Comment (tathian) [reply Rose/database/migrations/2014_10_12_000000_create_users_table.php]: I am doing search function so I added seed data

**Comment by nhanhvasnet** [comment]
Please read again Lavarel framework to know how to seed database. Thanks

**Pull request closed without merging**